### PR TITLE
Disable background manager by default.

### PR DESCRIPTION
**Pull request by cmeiklejohn**
@jrwest @kellymclaughlin

### CODE REVIEW

**Comment by kellymclaughlin** [comment]
Did a quick (hahaha) run of `replication2` and it is still passing. Eunit, eqc, dialyzer, and xref look good.

:+1:

**Comment by kellymclaughlin** [comment]
@borshop merge

File: priv/riak_repl.schema [hunk 1]
```diff
@@ -132,7 +132,7 @@
 {mapping, "mdc.fullsync.use_bg_manager", "riak_repl.fullsync_use_background_manager", [
     {datatype, {enum, [true, false]}},
     {level, advanced},
-    {default, true}
+    {default, false}
```
Comment (jrwest) [inline priv/riak_repl.schema:135]: fwiw, I noticed that the other bg manager cuttlefish mappings are using `{dataype, flag}`.

Comment (cmeiklejohn) [inline priv/riak_repl.schema:135]: @joedevivo Thoughts?

Comment (seancribbs) [inline priv/riak_repl.schema:135]: enum is still valid, no need to change it for RC

Comment (cmeiklejohn) [inline priv/riak_repl.schema:135]: Perfect, thanks.

Comment (jrwest) [inline priv/riak_repl.schema:135]: another thing i noticed -- also probably not needed before RC -- https://github.com/basho/riak_repl/blob/develop/test/riak_repl_schema_tests.erl does not test this mapping.

**Pull request merged**